### PR TITLE
feat: Added Homepage

### PR DESCRIPTION
**Pull request by parthgabani777**
- Added Homepage.
- Added Header Component.
- Added routes for all pages.
- Imported all CSS and images for project.

### CODE REVIEW

File: src/css/design.css [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+}
+.br-2{
+      border-radius: 10px;
+}
+.br-3{
+      border-radius: 15px;
+}
+.br-4{
+      border-radius: 50%;
+}
```
Comment (ajay-mandaviya) [inline src/css/design.css:116]: if you alredy made  it this class into your component library then you can use from your component no need to write it over here

**Review by ajay-mandaviya — APPROVED** [review]
Everything looks nice  ,  
make try to make your page folder like 
pages
       cart / cart.js && cart.css
       HomePage/ Homepage.js and homepage.css
      PageName / PageFile.js and pageFile.css

File: src/css/design.css (hunk 1 above)
Comment (parthgabani777) [reply src/css/design.css:116]: okay

File: src/App.js [hunk 2]
```diff
@@ -1,43 +1,32 @@
...
+            <BrowserRouter>
+                <Header />
+                <Routes>
+                    <Route path="/" element={<HomePage />} />
+                    <Route path="homePage" element={<HomePage />} />
+                    <Route path="login" element={<Login />} />
+                    <Route path="signup" element={<SignUp />} />
+                    <Route path="product" element={<Product />} />
+                    <Route path="wishlist" element={<WishList />} />
+                    <Route path="cart" element={<Cart />} />
```
Comment (prathmeshjagtap) [inline src/App.js:25]: you can add one more route to page not found 404 error

File: src/css/design.css [hunk 3]
```diff
@@ -0,0 +1,117 @@
+:root{
+      /* Colors */
+      /* #2b6777 #c8d8e4 #ffffff #f2f2f2 #52ab98 */
```
Comment (prathmeshjagtap) [inline src/css/design.css]: you can remove commented  code

**Review by prathmeshjagtap — APPROVED** [review]
Code is clean and well structured

File: src/App.js (hunk 2 above)
Comment (parthgabani777) [reply src/App.js:25]: I will add it in the upcoming PR.

**Pull request merged**